rust/README.md

```diff
@@ -0,0 +1,196 @@
+# ONNX Runtime
+
+These are Rust bindings to
+[Microsoft's ONNX Runtime](https://github.com/microsoft/onnxruntime).
+
+This project consists of two crates:
+
+* [`onnxruntime-sys`](onnxruntime-sys): Low-level binding to the C API;
+* [`onnxruntime`](onnxruntime): High-level and safe API.
+
+The `build.rs` script supports downloading pre-built versions of the Microsoft ONNX Runtime,
+which provides the following targets:
+
+CPU:
+
+* Linux x86_64
+* macOS x86_64
+* macOS aarch64
+* Windows i686
+* Windows x86_64
+
+GPU:
+
+* Linux x86_64
+* Windows x86_64
+
+---
+
+**WARNING**:
+
+* This is an experiment and work in progress; it is _not_ complete/working/safe. Help welcome!
+* Basic inference works, see [`onnxruntime/examples/sample.rs`](onnxruntime/examples/sample.rs) or [`onnxruntime/tests/integration_tests.rs`](onnxruntime/tests/integration_tests.rs)
+* ONNX Runtime has many options to control the inference process but those options are not yet exposed.
+
+---
+
+## Setup
+
+Three different strategy to obtain the ONNX Runtime are supported by the `build.rs` script:
+
+1. Download a pre-built binary from upstream;
+2. Point to a local version already installed;
+3. Compile from source.
+
+To select which strategy to use, set the `ORT_RUST_STRATEGY` environment variable to:
+
+1. `download`: Download prebuilt onnxruntime;
+2. `system`: To use a locally installed version (use `ORT_RUST_LIB_LOCATION` environment variable to point to the install path)
+3. `compile`: To compile the library. This is the default.
+
+The `download` strategy supports downloading a version of ONNX that supports CUDA. To use this, set the
```

Comment:
> ONNX t [](http://example.com/codeflow?start=57&length=7)

Here and in some other places ONNXRuntime. ONNX is project in a different depot.

Reply:
I will check for these and make sure ONNX -> ONNXRuntime